docs/content/en/docs/reference/changelog.md

```diff
@@ -6,12 +6,28 @@ weight: 35
 
 ## Unreleased
 
+### Added
+
+### Changed
+
+### Fixed
+
+## [v0.12.1](https://github.com/aws/eks-anywhere/releases/tag/v0.12.1)
+
+### Changed
+- Setting minimum wait time for nodes and machinedeployments (#3868, fixes #3822)
+
+### Fixed
+- Fixed worker node count pointer dereference issue (#3852)
+- Fixed eks-anywhere-packages reference in go.mod (#3902)
+- Surface dropped error in Cloudstack validations (#3832)
```

Comment:
shouldn't these be links?

Reply:
This is the changelog format we have standardized upon, inspired by https://keepachangelog.com. So just issues or PRs in parentheses, without hyperlink rendering.